packages/compass-import-export/test/setup.js

```diff
@@ -9,7 +9,9 @@ const chaiAsPromised = require('chai-as-promised');
 
 require('jsdom-global')();
 
+// eslint-disable-next-line no-undef
 window.Object = Object;
```

Comment:
I could change eslint config to allow `window` as global, but I prefer to add annotations instead